File: main/10_functions.py
## Functions
import pandas as pd
df = pd.read_csv('00_py_data_set.tsv', sep='\t')

def df_wrangler(df, value=10):
    import pandas as pd
    random_row = df.sample(n=1).reset_index()
    if random_row.at[0, 'Interesting']:
        print(random_row.at[0, 'Name'] + ' is interesting. BTW, you set the value as ' + str(value) + '.')
    else:
        print(random_row.at[0, 'Name'] + ' is not interesting. BTW, you set the value as ' + str(value) + '.')

df_wranlger(df, 7)

File: main/10_functions.R
## Functions
library(tidyverse)
df = read_tsv('00_R_data_set.tsv')

df_wrangler = function(df, value=10) {
  random_number = round(runif(n = 1, min = 1, max = nrow(df)))
  random_row = df[random_number,]
  if (random_row$Interesting) {
    return(paste(random_row$Name, ' is interesting. BTW, you set the value as ', value, '.', sep=''))
  } else {
    return(paste(random_row$Name, ' is not interesting. BTW, you set the value as ', value, '.', sep=''))
  }
}

df_wrangler(df, 7)
